src/javascript/app/pages/trade/tradepage.js

```diff
@@ -53,6 +54,20 @@ const TradePage = (() => {
                         TopUpVirtualPopup.init(State.getResponse('balance.balance'));
                     });
                 }
+            } else {
+                BinarySocket.send({ landing_company: 1 }).then(() => {
```

Comment:
I think we already send landing_company on page, so we can just wait here instead of sending again

Reply:
Ah that makes sense, thank you. 🙏